internal/pkg/manifest/validate.go

```diff
@@ -93,6 +101,21 @@ func (l LoadBalancedWebService) Validate() error {
 	return nil
 }
 
+func (d DeploymentConfiguration) Validate() error {
+	if d.isEmpty() {
+		return nil
+	}
+	for _, validStrategy := range deployConfigValidStrategies {
+		if strings.EqualFold(aws.StringValue(d.Rolling), validStrategy) {
+			return nil
+		}
+	}
+	return fmt.Errorf("invalid deployment strategy %s: %s %s",
+		aws.StringValue(d.Rolling),
+		english.PluralWord(len(deployConfigValidStrategies), "the valid deployment strategy is", "valid deployment strategies are"),
+		english.WordSeries(deployConfigValidStrategies, "and"))
```

Comment:
really nice! just some different wording:
```suggestion
	return fmt.Errorf("invalid deployment strategy %s, must be one of %s",
		aws.StringValue(d.Rolling),
		english.WordSeries(deployConfigValidStrategies, "or"))
```
I think it's fine to not check the length since we can never go back down to 1 after shipping

Reply:
make sense